### PR TITLE
Dev/backend workspace reorganization

### PR DESCRIPTION
**Pull request by burakcankazim**
Reorganized the workspace  to make future features easier to add. Moved classes about login feature to a new package. From now on each future should get its own package. And removed business logic from the controller layer.

### CODE REVIEW

File: application/backend/src/main/java/cmpe451/group12/beabee/login/controller/UserController.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
+package cmpe451.group12.beabee.login.controller;
```
Comment (veyisTurgut) [inline application/backend/src/main/java/cmpe451/group12/beabee/login/controller/UserController.java:1]: I didn't understand why did you move userController to login package. Isn't it supposed to be in user package?

**Comment by veyisTurgut** [comment]
Other than the issue above, I see that changes made according to our meeting.

File: application/backend/src/main/java/cmpe451/group12/beabee/login/controller/UserController.java (hunk 1 above)
Comment (burakcankazim) [reply application/backend/src/main/java/cmpe451/group12/beabee/login/controller/UserController.java:1]: Since we are going to use feature based package structure, the package that most sense to put the user related classes is login package, however we can change however we want.
